src/server/storage/memory.js

```diff
@@ -100,6 +101,69 @@ function Memory(mainLogger, gmeConfig) {
             return deferred.promise.nodeify(callback);
         };
 
+        this.loadPaths = function (rootHash, paths, excludes, callback) {

```

Comment:
Ok, no it's definitely better if we can keep the feature the same across the different implementations.
In general though, we should perhaps consider having either a base class or a set of utility functions for the "data-store" classes.